source/common/http/http1/codec_impl.cc

```diff
@@ -388,15 +388,19 @@ http_parser_settings ConnectionImpl::settings_{
       return static_cast<ConnectionImpl*>(parser->data)->onHeadersCompleteBase();
     },
     [](http_parser* parser, const char* at, size_t length) -> int {
-      static_cast<ConnectionImpl*>(parser->data)->onBody(at, length);
+      static_cast<ConnectionImpl*>(parser->data)->bufferBody(at, length);
       return 0;
     },
     [](http_parser* parser) -> int {
       static_cast<ConnectionImpl*>(parser->data)->onMessageCompleteBase();
       return 0;
     },
-    nullptr, // on_chunk_header
-    nullptr  // on_chunk_complete
+    [](http_parser* parser) -> int {
+      const bool is_final_chunk = (parser->content_length == 0);
```

Comment:
no no, content length is the length of the body, see?
https://github.com/nodejs/http-parser/blob/master/http_parser.h#L307

.....
reads further: https://github.com/nodejs/http-parser/blob/master/http_parser.h#L337
.....

never mind :-(
optionally, maybe comment this field is overloaded, in case other people have concerns?

Reply:
Added a comment.  Thanks!